spec/models/article_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+	subject { Article.new(title: "Great concepts of Rails")} 
```

Comment:
Use factories for building/creating your test data here and below.

Reply:
Using factory bot to build test data and db cleaner to destroy it back.
Thank you.